src/main/java/uk/gov/hmcts/reform/laubackend/idam/dto/UserDeletionUser.java

```diff
@@ -1,10 +1,33 @@
 package uk.gov.hmcts.reform.laubackend.idam.dto;
 
 
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import uk.gov.hmcts.reform.laubackend.idam.domain.UserDeletionAudit;
 
-public record UserDeletionUser(
-    String userId,
-    String emailAddress,
-    String firstName,
-    String lastName
-) { }
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Schema(description = "Data model for the IdAM user deletion request")
+public class UserDeletionUser {
+
+    @Schema(description = "The user who has been deleted")
+    String userId;
+    @Schema(description = "The email address of deleted user")
+    String emailAddress;
+    @Schema(description = "First name of deleted user")
+    String firstName;
+    @Schema(description = "Last name of deleted user")
+    String lastName;
+
+    public static UserDeletionUser toDto(final UserDeletionAudit userDeletionAudit) {
```

Comment:
Do we use UserDeletionUser.toDto() anywhere?

Reply:
This class (UserDeletionUser) is a leftover from initial model design. Expect it to go away once GET request is implemented. I left it in the codebase so that the get/search repository tests would not fail, once I add GET request endpoint I intend to remove this class.